### DISCUSSION TITLE
Help with integrating terminal into FOSS project

### DISCUSSION THREAD

**Discussion by N6REJ** [microsoft/terminal#14201] [General]
Who would we talk to to get some help in integrating windows terminal into our FOSS project, http://github.com/Bearsampp/Bearsampp
We are using "ConsoleZ" very heavily in our project but it hasn't been updated in many many years.
The pro's to it is its inherent ability to be the console for ruby, python, yarn, etc.  We'd like to replace it with windows terminal but are unclear how to go about it in a portable manor.

We understand that it would have to exist on the machine before it could be used, so we would make it part of our prerequisite package.

**Comment by zadjii-msft** [comment]
What's the min Windows version you're targeting/? The Windows Terminal comes pre-installed on all Windows 11 versions, but it's generally only available down to 19041. That'd be the first question that'd need to be answered

**Comment by N6REJ** [comment]
we are targeting windows 10+.  I suspect it will work on 8 but thats not our audience
